src/components/Header.js

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link'
+import ProfileIcon from './ProfileIcon'
```

Comment:
For consistency, we can use this. the `~` referring to `src` directory

```suggestion
import ProfileIcon from '~/components/ProfileIcon'
```

Reply:
Oke, nanti ane sesuaikan yang ini